SCons/Tool/swig.xml

```diff
@@ -126,22 +126,12 @@ construction variable.
 <cvar name="SWIGFLAGS">
 <summary>
 <para>
-General options passed to
-the scripting language wrapper and interface generator.
-This is where you should set
-<option>-python</option>,
+General options passed to &swig;.
+This is where you should set the target language
+(<option>-python</option>,
 <option>-perl5</option>,
-<option>-tcl</option>,
-or whatever other options you want to specify to SWIG.
-If you set the
-<option>-c++</option>
-option in this variable,
-&scons;
-will, by default,
-generate a C++ intermediate source file
-with the extension that is specified as the
-&cv-link-CXXFILESUFFIX;
-variable.
+<option>-tcl</option>, etc.)
+and whatever other options you want to specify to &swig;.
```

Comment:
Why remove the -c++ option from here?
Because it's discussed in the other options above?

Reply:
yes... c++ is a switch to change from generating c to generating c++; these here hare the languages you can generate the bindings *to*, so I felt separating them made sense.